README.md

```diff
@@ -67,18 +67,28 @@ If your build fails and you are unsure of why here are some useful strategies fo
 
 ## Common questions
 
-1. Why are my builds with plots failing on macOS?
+1. *Why are my builds with plots failing on macOS?*\
```

Comment:
I tried to harmonize the formatting for the questions - if the format should be different, please let me know.